### ISSUE TITLE
Validation command returns missleading command line output

### ISSUE THREAD

**Issue by fkromer** [doorstop-dev/doorstop#478]
With `doorstop==2.1.1` [some commands for setup are not working properly](https://github.com/doorstop-dev/doorstop/issues/442). I tried to create the doorstop files from scratch. I've the following structure:

```
$ tree reqs -a
reqs
├── .doorstop.yml
├── nodes
│   ├── .doorstop.yml
│   └── RN_001.yml
└── RP_001.yml
```

`./reqs/.doorstop.yml` content is:

```
settings:
    digits: 3
    prefix: RP
    sep: '_'
```

`./reqs/RP_001.yml` content is:

```
active: true
derived: false
header: ''
level: 1.1
links: []
normative: false
ref: ''
reviewed: null
text: |
  A pipeline **shall** do something.
```

`./reqs/nodes/.doorstop.yml` content is:

```
settings:
    digits: 3
    parent: RP
    prefix: RN
    sep: '_'
```

`./reqs/nodes/RN_001.yml` content is:

```
active: true
derived: false
header: ''
level: 1.2
links:
- RP_001: null
normative: false
ref: ''
reviewed: null
text: |
  A node **shall** do something.
```

There is not any string `HLT` in the overall project. However if I run the [validation command](https://doorstop.readthedocs.io/en/latest/cli/validation/#integrity-checks) I get the following output:

```
$ doorstop
building tree...
ERROR: unplaced document: HLT
```

Are there any constraints w.r.t. one has to setup the directory structure?

**Comment by jacebrowning** [comment]
Try `doorstop -vvv`. My guess is that `reqs/` is inside of another Doorstop project. Is the current directory under version control?

**Comment by fkromer** [comment]
@jacebrowning Thx a lot for the response. Seems like doorstop's own reqs in the virtual environment placed into the `<project-root>` is the root cause for the issue.

```
$ doorstop -vvv
[DEBUG   ] launching main command...
[INFO    ] looking for documents in /home/florian/gitlab/<project-root>...
[DEBUG   ] loading Document('/home/florian/gitlab/<project-root>/reqs')...
[TRACE   ] reading text from '/home/florian/gitlab/<project-root>/reqs/.doorstop.yml'...
[INFO    ] found document: RP (@/reqs)
[DEBUG   ] loading Document('/home/florian/gitlab/<project-root>/reqs/nodes')...
[TRACE   ] reading text from '/home/florian/gitlab/<project-root>/reqs/nodes/.doorstop.yml'...
[INFO    ] found document: RN (@/reqs/nodes)
[DEBUG   ] loading Document('/home/florian/gitlab/<project-root>/.venv/lib/python3.8/site-packages/doorstop/cli/tests/docs')...
[TRACE   ] reading text from '/home/florian/gitlab/<project-root>/.venv/lib/python3.8/site-packages/doorstop/cli/tests/docs/.doorstop.yml'...
[INFO    ] found document: HLT (@/.venv/lib/python3.8/site-packages/doorstop/cli/tests/docs)
[DEBUG   ] loading Document('/home/florian/gitlab/<project-root>/.venv/lib/python3.8/site-packages/doorstop/core/tests/docs')...
[TRACE   ] reading text from '/home/florian/gitlab/<project-root>/.venv/lib/python3.8/site-packages/doorstop/core/tests/docs/.doorstop.yml'...
[INFO    ] found document: LLT (@/.venv/lib/python3.8/site-packages/doorstop/core/tests/docs)
[DEBUG   ] loading Document('/home/florian/gitlab/<project-root>/.venv/lib/python3.8/site-packages/doorstop/core/tests/files')...
[TRACE   ] reading text from '/home/florian/gitlab/<project-root>/.venv/lib/python3.8/site-packages/doorstop/core/tests/files/.doorstop.yml'...
[DEBUG   ] skipped document: REQ (@/.venv/lib/python3.8/site-packages/doorstop/core/tests/files)
[DEBUG   ] loading Document('/home/florian/gitlab/<project-root>/.venv/lib/python3.8/site-packages/doorstop/core/tests/files/new')...
[TRACE   ] reading text from '/home/florian/gitlab/<project-root>/.venv/lib/python3.8/site-packages/doorstop/core/tests/files/new/.doorstop.yml'...
[DEBUG   ] skipped document: NEW (@/.venv/lib/python3.8/site-packages/doorstop/core/tests/files/new)
[DEBUG   ] loading Document('/home/florian/gitlab/<project-root>/.venv/lib/python3.8/site-packages/doorstop/core/tests/files/parent')...
[TRACE   ] reading text from '/home/florian/gitlab/<project-root>/.venv/lib/python3.8/site-packages/doorstop/core/tests/files/parent/.doorstop.yml'...
[DEBUG   ] skipped document: SYS (@/.venv/lib/python3.8/site-packages/doorstop/core/tests/files/parent)
[DEBUG   ] loading Document('/home/florian/gitlab/<project-root>/.venv/lib/python3.8/site-packages/doorstop/core/tests/files/child')...
[TRACE   ] reading text from '/home/florian/gitlab/<project-root>/.venv/lib/python3.8/site-packages/doorstop/core/tests/files/child/.doorstop.yml'...
[DEBUG   ] skipped document: TST (@/.venv/lib/python3.8/site-packages/doorstop/core/tests/files/child)
[INFO    ] building tree...
[INFO    ] root of the tree: RP (@/reqs)
[DEBUG   ] trying to add RN (@/reqs/nodes)...
[INFO    ] added to tree: RN (@/reqs/nodes)
[DEBUG   ] trying to add HLT (@/.venv/lib/python3.8/site-packages/doorstop/cli/tests/docs)...
[DEBUG   ] trying to add HLT (@/.venv/lib/python3.8/site-packages/doorstop/cli/tests/docs)...
[DEBUG   ] unknown parent for HLT (@/.venv/lib/python3.8/site-packages/doorstop/cli/tests/docs): TUT
[DEBUG   ] trying to add LLT (@/.venv/lib/python3.8/site-packages/doorstop/core/tests/docs)...
[DEBUG   ] trying to add LLT (@/.venv/lib/python3.8/site-packages/doorstop/core/tests/docs)...
[DEBUG   ] unknown parent for LLT (@/.venv/lib/python3.8/site-packages/doorstop/core/tests/docs): REQ
[DEBUG   ] trying to add HLT (@/.venv/lib/python3.8/site-packages/doorstop/cli/tests/docs)...
[DEBUG   ] trying to add HLT (@/.venv/lib/python3.8/site-packages/doorstop/cli/tests/docs)...
[DEBUG   ] unknown parent for HLT (@/.venv/lib/python3.8/site-packages/doorstop/cli/tests/docs): TUT
[DEBUG   ] trying to add LLT (@/.venv/lib/python3.8/site-packages/doorstop/core/tests/docs)...
[DEBUG   ] trying to add LLT (@/.venv/lib/python3.8/site-packages/doorstop/core/tests/docs)...
[DEBUG   ] unknown parent for LLT (@/.venv/lib/python3.8/site-packages/doorstop/core/tests/docs): REQ
[DEBUG   ] unplaced documents: [Document('/home/florian/gitlab/<project-root>/.venv/lib/python3.8/site-packages/doorstop/cli/tests/docs'), Document('/home/florian/gitlab/<project-root>/.venv/lib/python3.8/site-packages/doorstop/core/tests/docs')]
[ERROR   ] unplaced document: HLT (@/.venv/lib/python3.8/site-packages/doorstop/cli/tests/docs)
[DEBUG   ] command failed
```

Is there a way to configure to ignore this directory?

**Comment by fkromer** [comment]
Usually `.doorstop.skip` needs to be placed into the directory to be ignored, right?

**Comment by jacebrowning** [comment]
This should be fixed in v2.1.2.

**Comment by fkromer** [comment]
@jacebrowning Great. Thanks a lot.
